iree/tools/init_translations.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file defines a helper to trigger the registration of all translations
+// in and out of MLIR to the system.
+//
+// Based on MLIR's InitAllTranslations but without translations we don't care
+// about.
+
+#ifndef IREE_TOOLS_INIT_TRANSLATIONS_H_
+#define IREE_TOOLS_INIT_TRANSLATIONS_H_
+
+namespace mlir {
+
+void registerFromLLVMIRTranslation();
+void registerToLLVMIRTranslation();
```

Comment:
I am actually not 100% sure if we really need to `register{To,From}LLVMIRTranslation()` functions. Comparing the CMakeLists, we're only linking in `MLIRSPIRVSerialization` with `ALWAYSLINK`, but do not depend on `MLIRTargetLLVMIR`. So probably these two register functions can be removed. Can somebody confirm?

Reply:
I don't think anything should be using these. The LLVM target we have is jiting, so it should be happening at runtime, I think.

@MaheshRavishankar @asaadaldien can you confirm?